precompiles/vesting/types.go

```diff
@@ -259,3 +259,19 @@ func (bo *BalancesOutput) FromResponse(res *vestingtypes.QueryBalancesResponse)
 func (bo *BalancesOutput) Pack(args abi.Arguments) ([]byte, error) {
 	return args.Pack(bo.Locked, bo.Unvested, bo.Vested)
 }
+
+// ClawbackOutput represents the clawed back coins from a Clawback transaction.
+type ClawbackOutput struct {
+	Coins []cmn.Coin
+}
+
+// FromResponse populates the ClawbackOutput from a QueryClawbackResponse.
+func (co *ClawbackOutput) FromResponse(res *vestingtypes.MsgClawbackResponse) *ClawbackOutput {
+	co.Coins = cmn.NewCoinsResponse(res.Coins)
+	return co
+}
+
+// Pack packs a given slice of abi arguments into a byte array.
+func (co *ClawbackOutput) Pack(args abi.Arguments) ([]byte, error) {
+	return args.Pack(co.Coins)
```

Comment:
nit: this isn't used anywhere, you can call it directly with `out.Pack(method.Outputs)` or remove it.

Reply:
good point, I was wondering about this logic for the Balances query as well - I'll remove it there, too!